apps/zui/package.json

```diff
@@ -91,7 +91,7 @@
     "electron-localshortcut": "^3.2.1",
     "electron-log": "5.0.0-beta.6",
     "electron-mock-ipc": "^0.3.12",
-    "electron-updater": "^4.3.8",
+    "electron-updater": "6.2.1",
```

Comment:
Before we were using the `^` to automatically update to minor/patch `electron-updater` releases as they became available. I've now spent enough time testing through the release history of `electron-updater` that I don't wholly trust the stewardship of the project to maintain stability between major releases. We also don't have automated test coverage in this area, which is more risk. Therefore since I've done so many days of testing on this `6.2.1` release I'm keen to lock us in place until we have a specific reason to move to something newer and make the appropriate investment in another round of testing at that time.